backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt

```diff
@@ -101,7 +101,7 @@ class InfluxDbProxy {
                 + " |> range(start: $start, stop: $end)"
                 + " |> filter(fn: (r) => (" +
                     "r._measurement == \"host_stats\" and " +
-                    "r.host_id == \"$hostId.\" and " +
+                    "r.host_id == \"$hostId\" and " +
```

Comment:
FIX ALERTS >:(

Reply:
alerts are fixed on another branch. No need to fix them on right here right now

Reply:
i'll fix your face